src/70insert.js

```diff
@@ -331,6 +335,75 @@ yy.Insert.prototype.compile = function (databaseid) {
 	} else if (this.default) {
 		var insertfns = "db.tables['" + tableid + "'].data.push({" + table.defaultfns + '});return 1;';
 		var insertfn = new Function('db,params,alasql', insertfns);
+	} else if (this.setcolumns) {
```

Comment:
This is a log of code. Are you sure this is the best place to do this, and is there a better way to make the code more slim?